graphene/pyutils/version.py

```diff
@@ -44,8 +44,10 @@ def get_complete_version(version=None):
     if version is None:
         from graphene import VERSION as version
     else:
-        assert len(version) == 5
```

Comment:
I suggest only using `is` for comparing versus None, or identity comparisons. `==` is for comparing logical equivalence, which is going to be pretty much all numeric comparisons.

Some of the answers on https://stackoverflow.com/questions/2239737/is-it-better-to-use-is-or-for-number-comparison-in-python discuss it a bit more (1st SO page I googled).